supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesMetadataKeys.scala

```diff
@@ -5,8 +5,9 @@ object JesMetadataKeys {
   val GoogleProject = "jes:googleProject"
   val ExecutionBucket = "jes:executionBucket"
   val EndpointUrl = "jes:endpointUrl"
-  val MonitoringLog = "jes:monitoringLog"
+  val MonitoringScript = "jes:monitoringScript"
   val MachineType = "jes:machineType"
   val Zone = "jes:zone"
   val InstanceName = "jes:instanceName"
+  val MonitoringLog = "monitoringLog"
```

Comment:
Mentioned this offline but #2507 asks for this to be across all backends, but for now it's just JES